cns/api.go

```diff
@@ -59,6 +61,7 @@ type IPConfigsHandlerFunc func(context.Context, IPConfigsRequest) (*IPConfigsRes
 // IPConfigsHandlerMiddleware
 type IPConfigsHandlerMiddleware interface {
 	IPConfigsRequestHandlerWrapper(defaultHandler IPConfigsHandlerFunc, failureHandler IPConfigsHandlerFunc) IPConfigsHandlerFunc
+	GetMiddlewareType() configuration.SWIFTV2Mode
```

Comment:
I think making this `Type()` would be sufficiently verbose here?

Reply:
what is the usual pattern in cns/cni repo? I mapped from our internal repo dnc to atleast use a GetType() kind of name to make the intent explicit in caller function

Reply:
@kmurudi in this case, I think @nddq is correct. You have to consider the entire signature in the name. Since this can only return a `configuration.SWIFTV2Mode`, then `Type()` is reasonably concise. If ever there is a question of "what kind of Type is it?" you can refer to the return type.